doc/particles.md

```diff
@@ -5,85 +5,88 @@ Flame offers a basic, yet robust and extendable particle system. The core concep
 The most basic usage of `Particle` with `BaseGame` would look as following:
 
 ```dart
-import 'package:flame/components/particle_component.dart';
+import 'package:flame/components.dart';
 
 // ... 
 
 game.add(
-    // Wrapping a [Particle] with [ParticleComponent]
-    // which maps [Component] lifecycle hooks to [Particle] ones
-    // and embeds a trigger for removing the component.
+    /// Wrapping a [Particle] with [ParticleComponent]
+    /// which maps [Component] lifecycle hooks to [Particle] ones
+    /// and embeds a trigger for removing the component.
     ParticleComponent(
         particle: CircleParticle()
     )
 );
 ```
 
-When using `Particle` with custom `Game` implementation, please ensure that `Particle` `update` and `render` lifecycle hooks are called during each game loop frame.
+When using `Particle` with a custom `Game` implementation, please ensure that both the `update` and `render` methods are called during each game loop tick.
```

Comment:
Make sure that the max line length is 100 in this file too

Reply:
I just cant seem to get my IDE to format it the way I want it

Reply:
Just do a line break manually?

Reply:
I'm using the rewrap plugin for vscode, it's really nice!